### DISCUSSION TITLE
how to close Object Locking and  Version？

### DISCUSSION THREAD

**Discussion by blog2023sunshine** [minio/minio#19338] [Q&A] [answered]
I create a bucket and enable Object Locking and  Version。
Afterwards, a lot of data was placed in the bucket
now I want disable Object Locking and  Version ，bug i failed。
Dear experts, can you help me？   (◒｡◒) 

D:\>mc retention clear --default   minio/test-del
Object lock configuration cleared successfully.

D:\>mc  version  suspend   minio/test-del
mc: <ERROR> Unable to suspend versioning. An Object Lock configuration is present on this bucket, versioning cannot be suspended.

**Comment by harshavardhana** [comment, marked as answer]
You cannot disable object locking once it is enabled for a bucket, you can only clear retention settings. You cannot disable versioning on an object-locked bucket.

**Reply by blog2023sunshine** [reply]
Thank you very much for your reply
